cpu/kinetis/periph/spi.c

```diff
@@ -136,7 +136,7 @@ int spi_init_cs(spi_t bus, spi_cs_t cs)
     }
     else {
         if ((cs >= SPI_HWCS_NUMOF) ||
-            (spi_config[bus].pin_cs[cs] == GPIO_UNDEF)) {
+            (spi_config[bus].pin_cs[cs] == SPI_CS_UNDEF)) {
```

Comment:
```suggestion
            (gpio_is_valid(spi_config[bus].pin_cs[cs])) {
```

Reply:
This is an equal. So it should rather be `!gpio_is_valid`

Reply:
> This is an equal. So it should rather be `!gpio_is_valid`

Yeah, sure.